bin/dot-only-hunter

```diff
@@ -47,12 +47,18 @@ main () {
 
   echo "Hunting inside '$test_dir' for tests with \`.only\`..."
 
-  if grep -nR "\($prey\)\.only" "$test_dir"; then
+  grep -nR "\($prey\)\.only" "$test_dir"
+  rc=$?
+  if [[ $rc -eq 0 ]]; then
     echo "Whoops! Found \`.only\` in your tests."
     exit 1
+  elif [[ $rc -gt 1 ]] ; then
+    echo "Error searching for\`.only\` in your tests. This system may be incompatible."
```

Comment:
I'd prefer "Unexpected `grep` error. Please report this issue! :)\nhttps://github.com/dasilvacontin/dot-only-hunter/issues/new".

Once we make `-R` gracefully fallback to `-r`, all other `grep` errors are unexpected.